packages/airnode-node/src/handlers/process-http-request.ts

```diff
@@ -12,7 +11,6 @@ export async function processHttpRequest(
   parameters: Record<string, string>
 ): Promise<[Error, null] | [null, ApiCallSuccessResponse]> {
   const requestId = randomHexString(16);
-  const airnodeAddress = wallet.getAirnodeWallet(config).address;
   const logOptions = buildBaseOptions(config, { requestId });
```

Comment:
Since requestId is optional in LogMetadata, does it make sense to add a random hex string to the log options in this case?

Reply:
I think you're right, it doesn't appear strictly necessary in isolation, but `processHttpSignedDataRequest` generates the `requestId` in the same (random) way and uses that `id` elsewhere in an error message:
https://github.com/api3dao/airnode/blob/62a090eddf37db93ebc64ba10ec70f21199c4dbe/packages/airnode-node/src/api/index.ts#L138-L139

So I'd lean towards keeping it as it also feels right to have a unique Id within the logs for a request, but if you'd like I can remove it.

Reply:
Yes it makes sense but this will force us to submit a random hex in the payload from Airkeeper to only be used in those logs where [we are not sure if we should even be calling that function for beacons](https://github.com/api3dao/airnode/blob/62a090eddf37db93ebc64ba10ec70f21199c4dbe/packages/airnode-node/src/api/index.ts#L129) 😁. 

I think you are right and we should keep it just to avoid any error but I also wanted to ask you about it because to me a `requestId` is a hash computed on chain for AirnodeRrp and I'm not sure if it applies to http-gateway, signed or Airkeeper requests. 

Thanks for investigating tho 👏🏻

Reply:
You mean calling from airkeeper [here](https://github.com/api3dao/airkeeper/blob/a6090d06facc60c16d60b4cbd8f0613c762dc261/src/api/call-api.ts#L12-L27)? In that case the payload won't need `id`, `airnodeAddress`, or `endpointId` anymore because [these properties are being removed from `BaseAggregatedApiCall`](https://github.com/api3dao/airnode/pull/996/files#diff-bafef992d17b51ddeeecf5c4f67a79342f80c0b28bd383c18310ce0da1a57278L226) and [`HttpGatewayAggregatedApiCall` doesn't have them](https://github.com/api3dao/airnode/blob/62a090eddf37db93ebc64ba10ec70f21199c4dbe/packages/airnode-node/src/types.ts#L258-L260). That is the point of the PR 😅